pyvista/_plot.py

```diff
@@ -8,45 +8,54 @@
 
 """
 
+# ruff: noqa: FBT001,FBT002
```

Comment:
To make it easier to identify which parameter is causing the error, please set it to asis.